buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -12,9 +12,9 @@ object Versions {
     const val ktor = "1.5.1"
     const val serialization = "1.0.1"
     const val micrometer = "1.6.3"
-    const val mySql = "8.0.20"
+    const val mySql = "8.0.22"
```

Comment:
Our database on the server is 8.0.20, so we should keep the same version here

Reply:
Fixed